src/components/Filters/index.tsx

```diff
@@ -111,25 +111,43 @@ class Filters extends React.Component<FiltersProps, FiltersState> {
 		const { filters, schema, views } = this.props;
 
 		const flatSchema = SchemaSieve.flattenSchema(schema);
+		const flatViews = this.flattenViews(views || []);
+		const flatFilters = (filters || []).map(filter =>
```

Comment:
If you like, you can actually just put defaults like this in the deconstruction above:

```ts
const { filters = [], schema, views = [] } = this.props;

...
```